Client/Frontend/ContentBlocker/TrackingProtectionPageStats.swift

```diff
@@ -64,7 +64,17 @@ class TPStatsBlocklistChecker {
         }
         
         DispatchQueue.global().async {
-            let enabledLists = BlocklistName.blocklists(forDomain: domain).on
+            var domainOnBackgroundContext: Domain?
+            var enabledLists = Set<BlocklistName>()
+            
+            DataController.performBackgroundTask { context in
```

Comment:
Given that these lists are only _read_ (I don't see any manipulation of them), and we don't even return the `Domain` that is pulled, I think using the `viewContext` is probably fine. Or do you think this is better since it doesn't _need_ to be on main thread?

Reply:
Yeah we can also use view context inside of `performAndWait` block, otherwise it throws an error. Not sure what's better in this case